cryptus-app/components/wallet_viewer/collection_details/collection_viewer/collection_viewer.tsx

```diff
@@ -0,0 +1,24 @@
+import React, { useEffect, useState } from "react";
+import s from "./collection_viewer.module.scss";
+
+import NftBlock from "./nft_block/nft_block"
+
+//Need to add prop for collection
+export default function CollectionViewer() {
+  const collection = [
+    {name: "CryptoPunk #9998", nft:"images/cryptopunk.png", id:"1"},
+    {name: "CryptoPunk #9998", nft:"images/cryptopunk.png", id:"2"},
+    {name: "CryptoPunk #9998", nft:"images/cryptopunk.png", id:"3"},
+    {name: "CryptoPunk #9998", nft:"images/cryptopunk.png", id:"4"},
+    {name: "CryptoPunk #9998", nft:"images/cryptopunk.png", id:"5"},
+  ];
```

Comment:
get collection from props instead, and code this test collection at the place where the backend call would be made